src/rabbit_mnesia.erl

```diff
@@ -596,21 +597,21 @@ check_cluster_consistency(Node, CheckNodesConsistency) ->
     case rpc:call(Node, rabbit_mnesia, node_info, []) of
         {badrpc, _Reason} ->
             {error, not_found};
-        {_OTP, _Rabbit, {error, _}} ->
+        {_OTP, Rabbit, Hash, _Status} when is_binary(Hash) ->
+            %% delegate hash checking implies version mismatch
```

Comment:
In previous versions (long time ago), `rabbit_mnesia:node_info` was returning 4-element tuple. 3rd element was `delegate_beam_hash` - beam md5 hash of `delegate` module. 
This commit adds 4-element tuple back, where 3rd element would be protocol consistency. Type match is here to avoid confusing one with another.